ankidroid/Anki-Android
# Color of the Bottom Show Answer Changes based on the theme color

## Fixes
* Fixes #14209


## Approach
From the theme changed the color of the button also replaced the hardcoded color code to theme color code using `@color/theme_xxx` so in future if the color code of the theme if changes then it will also get changed dynamically.

`<solid android:color="@color/theme_black_primary" />`

## Testing Instructions
 1. Change the theme of the app from (Hamburger Menu ---> Settings ---> Appearance ---> Select Theme)
 2. Choose the theme (Light, Plain, Black, Dark)
 3. Go to home. 
 4. Open any card and then check the show answer button color matches the theme of the app.
 5. Repeat step 1 till you test all theme (Light, Plain, Black, Dark)
 

## How Has This Been Tested?
This has been tested on physical device(Realme 3i) and on emulator (Google Pixel 3)

## Screenshots

### Light Mode
<img src="https://github.com/ankidroid/Anki-Android/assets/60827173/e0c9d528-4673-41cb-906a-33e08aeb5dca" width="50%" height="50%">

### Dark Mode

<img src="https://github.com/ankidroid/Anki-Android/assets/60827173/1e95dab7-37ae-4936-a0ef-842695d0b2f2" width="50%" height="50%">

### Plain Mode

<img src="https://github.com/ankidroid/Anki-Android/assets/60827173/0f0ffb0f-6053-4ee6-9017-56a3a8b83133" width="50%" height="50%">

### Black Mode

<img src="https://github.com/ankidroid/Anki-Android/assets/60827173/84bbca6c-0f79-4bd3-86ce-ca668143ed47" width="50%" height="50%">





## Checklist
